### ISSUE TITLE
Does not work, but worked before

### ISSUE THREAD

**Issue by yermak** [yermak/AudioBookConverter#489]
I see many bugs reports - "it does not work", with no details given.
Unfortunately, I can't help you in such case.

As a last workaround, I could advise (for Windows users):
Go to Application Data folder, for me it it's located "C:\Users\Yermak\Application Data" and DELETE folder
AudioBookConverter-x.y.z
where x.y.z - is version you are using, the latest one is 6.1.2,
So, you need to delete AudioBookConverer-6.1.2, this will reset program settings to defaults.

**Comment by yermak** [comment]
@thew1se

**Comment by yermak** [comment]
@ChelseaThomas86 
@Marvin-D-Martian

**Comment by yermak** [comment]
@NegativeReenforcement

**Comment by Marvin-D-Martian** [comment]
Hi yermak.
Sorry for the incomplete report. It was out of frustration.
I am using the latest version 6.1.2 . Even uninstalled and bought the Steam shop version. Same exact result.
I load the mp3 (no difference between adding a file or folder), it is recognised but when I hit start, the button highlights but nothing happens. The dialog asking me where to save just never appears. I can smash the button and nothing happens.

I installed a fresh copy in a VM environment and it worked, but eventually the same thing happened again and it just stops working. I used a snapshot and went to a moment when it still worked and then managed to convert 10 other books. So I am not sure if it's a rondom issue or some specific mp3 or file issue that breaks it... 
Sorry for not being able to give more details.
Your app is awesome, well worth VM shenanigans to be able to use it ;) Thanks!

**Comment by yermak** [comment]
@Marvin-D-Martian, did you try to remove AudioBookConverter folder in "Application Data" folder. But before you deleted it, could you please have a look into that folder and find a file AudioBookConverter.log, if you see obvious error message please post it here, alternatively send it to "my_github_name"@gmail.com, my email is same as github name. I will have a look.

**Comment by ChelseaThomas86** [comment]
I can give more details.

I open the program.
File.. Add... Files...
Click on my audio book file (that is in the same folder at the .cue file) 
         I got these audio book files with the Libby download extension on Firefox
I click on the chapters tab and see the chapters
I click Start and nothing happens
I click start a bunch more... still nothing. 

I am running Windows 11.

I will try the thing you suggested above.

**Comment by Marvin-D-Martian** [comment]
Hi. Many of the same repeating, sanitised a bit:
`25-08-2023 10:32:18 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
25-08-2023 10:32:18 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
`
`25-08-2023 10:32:37 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
25-08-2023 10:32:38 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
25-08-2023 10:32:38 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
25-08-2023 10:32:38 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
25-08-2023 10:32:42 INFO  AudiobookConverter:382 - Closing application
`
After I deleted the folder.

After first start works again so far, before I did anything, log already shows (even when ran as admin):
`
26-08-2023 19:15:33 INFO  AudiobookConverter:382 - Initialising application
26-08-2023 19:15:33 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:34 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:34 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:34 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:34 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:34 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:34 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:34 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:34 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:34 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\Steve\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:35 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:35 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:35 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:35 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:35 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:35 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:35 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:35 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:35 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:35 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:35 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:35 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
26-08-2023 19:15:35 WARN  FileDataWriter:195 - Can't open directory channel. Log directory fsync won't be performed.
26-08-2023 19:15:35 INFO  EnvironmentImpl:382 - Exodus environment created: C:\Users\User\AppData\Roaming\AudiobookConverter-6.1.2
`

Hope this helps. Many Thanks!
